packages/sdks/core-js-sdk/src/httpClient/helpers/createFetchLogger.ts

```diff
@@ -91,7 +91,13 @@ const fetchWrapper =
     const respText = await resp.text();
 
     resp.text = () => Promise.resolve(respText);
-    resp.json = () => Promise.resolve(JSON.parse(respText));
+    resp.json = () => Promise.resolve(() => {
```

Comment:
@meirwah  Could it be that due to Cloudflare rate limiting, we’re receiving HTML responses from our APIs?